swap_meet/vendor.py

```diff
@@ -1,2 +1,75 @@
+from .item import Item
+
 class Vendor:
-    pass
+    """Instantiates an object Vendor with an inventory.  Methods are adding or removing items from inventory,
+    retrive items by specified category, swap items with another vendor, swap first in inventory item with first item 
+    in inventory of another vendor, swap best item in specified category with the best item in specified category of 
+    another vendor.
+    """
+
+    def __init__(self, inventory=None):
+        """Instantiates an instance with an attribute inventory, by default an empty list"""
+        self.inventory = inventory if inventory else []
+    
+    def add(self, new_item):
+        """Adds new item to inventory list"""
+        self.inventory.append(new_item)
+        return new_item
+    
+    def remove(self, item):
+        """Removes specified item from inventory list"""
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
+    
+    def get_by_category(self, category):
+        """Returns all items in ventory with a specified category"""
+        category_items = []
+        for item in self.inventory:
+            if item.category == category:
+                category_items.append(item)
+        return category_items
+
+    def swap_items(self, swapie_vendor, item_going_out, item_coming_in):
+        """Swaps specified items between one instance of Vendor and another instance of Vendor"""
+        if item_going_out in self.inventory and item_coming_in in swapie_vendor.inventory:
+            swapie_vendor.add(item_going_out)
+            self.remove(item_going_out)
+            self.add(item_coming_in)
+            swapie_vendor.remove(item_coming_in)
+            return True
+        else:
+            return False
```

Comment:
Minor note.  Since the `if` block ends in a return, you don't need the else.  Some of the following methods do this as well. 

```suggestion
        return False
```